code-challenges.js

```diff
@@ -9,60 +9,186 @@
-var temp3 = 212
+// var temp1 = 35
+// var temp2 = 350
+// var temp3 = 212
 
+// const boilingPoint = (num) => {
+//     if(num > 212){
+//         return `${num} is above boiling point`
+//     } else if (num < 212){
+//         return `${num} is below boiling point`
+//     } else if (num == 212) {
+//         return `${num} is at boiling point`
+//     } else {
+//         return "something went wrong"
+//     }
+// }
+
+// console.log(boilingPoint(temp1));
+// console.log(boilingPoint(temp2));
+// console.log(boilingPoint(temp3));
 
 
 
 
 // --------------------2) Create a function that multiplies each number in the array by 5 using a for loop.
 // Use the test variable provided below. Expected outcome: [15, 35, 0, 30, -45]
 
-var myNumbers1 = [3, 7, 0, 6, -9]
-
+// var myNumbers1 = [3, 7, 0, 6, -9]
+// var newNumbers = []
 
+// const multiplier = (array) => {
+//     for (let index = 0; index < array.length; index++) {
+//         newNumbers.push(array[index] * 5)
+//     }
+//     return newNumbers
+// }
 
+// console.log(multiplier(myNumbers1));
 
 
 // --------------------3) Create a function that multiplies each number in the array by 5 using map.
 // Use the test variable provided below. Expected outcome: [40, -35, 0, 30, 10]
 
-var myNumbers2 = [8, -7, 0, 6, 2]
-
+// var myNumbers2 = [8, -7, 0, 6, 2]
+// var newArray = []
 
+// const multiplier = (array) => {
+//     newArray = array.map( value => {
+//         return value * 5
+//     })
+//     return newArray
+// }
 
+// console.log(multiplier(myNumbers2));
 
 
 // --------------------4) Create a function that removes all the vowels from a string.
 // Use the test variables provided below. Expected output: "HyThrLrnStdnt" "LvJvScrpt"
 
-var stringWithVowels1 = "HeyThereLearnStudent"
-var stringWithVowels2 = "ILoveJavaScript"
+// var stringWithVowels1 = "HeyThereLearnStudent"
+// var stringWithVowels2 = "ILoveJavaScript"
+// var newString = []
 
+// const vowelRemover = (string) => {
+//     let = newArray = []
+//     newString = string.split("")
+//     newArray = newString.filter(value => {
+//         return value != 'a' && value != 'e' && value != 'i' && value != 'o' && value != 'u' &&value != 'y'
+//     })
+//     return newArray.join("")
+// }
 
+// console.log(vowelRemover(stringWithVowels1));
+// console.log(vowelRemover(stringWithVowels2));
 
 
 
 // --------------------5) Copy/paste your code from #4. Refactor your code to include non-string edge cases. Inform your user that the variable passed into the vowel removal function is not a string.
 // Use the test variables provided below. Expected output: "true is not a string" "42 is not a string"
 
-var notAString1 = true
-var notAString2 = 42
+// var notAString1 = true
+// var notAString2 = 42
 
+// var stringWithVowels1 = "HeyThereLearnStudent"
+// var stringWithVowels2 = "ILoveJavaScript"
+// var newString = []
+// var stringCheck = ''
 
+// const vowelRemover = (string) => {
+//     stringCheck = typeof string
```

Comment:
no need to declare `stringCheck` above, you can just do it for the first time here